rkhunter/osfamilymap.yaml

```diff
@@ -39,143 +31,126 @@ Debian:
       - /usr/sbin/adduser
 
 RedHat:
-  package: rkhunter
-  default_file: /etc/sysconfig/rkhunter
-  config_file: /etc/rkhunter.conf
   default:
-    mailto: root@localhost
     diag_scan: 'no'
+    mailto: root@localhost
   config:
-    tmpdir: /var/lib/rkhunter
-    dbdir: /var/lib/rkhunter/db
-    scriptdir: /usr/share/rkhunter/scripts
-    logfile: /var/log/rkhunter/rkhunter.log
-    append_log: 1
-    auto_x_detect: 1
-    allow_ssh_root_user: unset
-    allow_ssh_prot_v1: 2
-    enable_tests: ALL
-    disable_tests:
-      - suspscan
-      - hidden_procs
-      - deleted_files
-      - packet_cap_apps
-      - apps
-      - ipc_shared_mem
-    pkgmgr: RPM
-    existwhitelist:
-      - /bin/ad
-      - /var/log/pki-ca/system
-      - /var/log/pki/pki-tomcat/ca/system
-      - /usr/bin/GET
-      - /usr/bin/whatis
-      - /var/log/pki/pki-tomcat/kra/system
-    scriptwhitelist:
-      - /usr/bin/whatis
-      - /usr/bin/ldd
-      - /usr/bin/groups
-      - /usr/bin/GET
-      - /sbin/ifup
-      - /sbin/ifdown
+    allowdevfile:
+      - "/dev/md/autorebuild.pid"
```

Comment:
@n-rodriguez By the way, I was planning to remove these quotation marks from the paths, to be consistent throughout, but I thought I'd ask you about them first.

Reply:
Done.